### PR TITLE
Added GetClientScope Method

### PR DESCRIPTION
**Pull request by manju754**

### CODE REVIEW

**Comment by manju754** [comment]
@SVilgelm  Can you review this PR

**Comment by Nerzal** [comment]
Hi, thanks for the PR.
Could you please also add the Method to the readme?

File: gocloak.go [hunk 1]
```diff
@@ -75,6 +75,8 @@ type GoCloak interface {
 
 	// GetClient returns a client
 	GetClient(accessToken string, realm string, clientID string) (*Client, error)
+	// GetClientScope returns a clientscope
+	GetClientScope(token string, realm string, scopeID string) (*ClientScope, error)
```
Comment (SVilgelm) [inline gocloak.go]: Please add this function to README.md

Comment (manju754) [reply gocloak.go]: Added

**Pull request merged**